### PR TITLE
Disabled logging late in enclave termination

### PR DESCRIPTION
**Pull request by anakrish**
Signed-off-by: Anand Krishnamoorthi <anakrish@microsoft.com>

### CODE REVIEW

File: enclave/core/sgx/calls.c [hunk 1]
```diff
@@ -453,7 +453,10 @@ static oe_result_t _enclave_destructor(void)
         /* Cleanup the allocator */
         oe_allocator_cleanup();
 
-        _destructor_done = true;
```
Comment (mingweishih) [inline enclave/core/sgx/calls.c:456]: why this is removed?

File: enclave/core/tracee.c [hunk 2]
```diff
@@ -52,6 +53,11 @@ void oe_log_init_ecall(const char* enclave_path, uint32_t log_level)
 {
     const char* filename;
 
+    // Beyoned a certain point in enclave termination, logging is disabled and
+    // logs are silenced.
+    if (_logging_disabled)
```
Comment (mingweishih) [inline enclave/core/tracee.c:58]: Is it necessary to block this ecall as well?

File: enclave/core/sgx/calls.c (hunk 1 above)
Comment (anakrish) [reply enclave/core/sgx/calls.c:456]: Accidental. Will revert.

**Comment by anakrish** [comment]
The original issue is not reproducible. Closing this PR.

**Pull request closed without merging**